extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientConfig.java

```diff
@@ -200,7 +200,6 @@ public class RestClientConfig {
     /**
      * If set to false disables the keep alive completely.
      */
-    @ConfigItem(defaultValue = "true")
```

Comment:
No idea if this change is right but you should keep the `@ConfigItem`, just remove the default value.

Reply:
Ay, yes, you're right.

Reply:
Am I right to think that this will change the behavior and default keepalive to false? Or is the true default set elsewhere?

Reply:
It just removes the default value for config-key/classname config values because it would overwrite any globally set config value due to the way the resolution is implemented.

Reply:
OK. I will ask the expert to review :). Could you squash your two commits? (If you know how to do it, otherwise I can do it).

We are using merge commits and we cannot squash on merge with this setting. Thanks!

Reply:
Yes I will take care of the squash. Thanks!